src/drivers/nuopc/cime_driver/esmApp.F90

```diff
@@ -129,7 +147,7 @@ program esmApp
        call ESMF_Finalize(endflag=ESMF_END_ABORT)
 
   call ESMF_LogWrite("ESMF_GridCompDestroy called", ESMF_LOGMSG_INFO, rc=rc)
-!  call ESMF_LogSet(flush=.true., trace=.true., rc=rc)
+  ! call ESMF_LogSet(flush=.true., trace=.true., rc=rc)
```

Comment:
why is this line commented out?

Reply:
I have no idea - I have not changed this - should it not have a comment?